Gallery/Contours/NCL_ce_3_1.py

```diff
@@ -4,27 +4,27 @@
 
 This script illustrates the following concepts:
    - Drawing color-filled contours over a cylindrical equi-distant map
-   - Selecting a different color map
    - Changing the contour level spacing
    - Turning off contour lines
-   - Comparing styles of map tickmarks labels
-   - Changing the stride of the colorbar labels
    - Zooming in on a particular area on the map
-   - Turning off the addition of a longitude cyclic point
 
 See following URLs to see the reproduced NCL plot & script:
     - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/ce_3.ncl
     - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/ce_3_1_lg.png
+
+Note:
```

Comment:
```suggestion
.. Warning::
```
If you use this header, you can make a nicely formatting warning in an orange box. It's really hard to miss and would be good to use here.